src/Server.ts

```diff
@@ -24,7 +34,7 @@ console.log(err);
 }
 console.log(`App is running on port ${PORT}`);
 // tslint:disable-next-line: semicolon
```

Comment:
Remove this lint rule disable.

Reply:
By mistake i forget to remove this lint rule but i resloved  it.